functions/server/fn_init.sqf

```diff
@@ -6,31 +6,25 @@ GRAD_REPLAY_PLAYBACK_PAUSED = false;
 
 if (!isServer) exitWith {};
 
-params [["_precision", 1]];
-
 // constants
 GRAD_REPLAY_RECORDING_PAUSED = false;
 GRAD_REPLAY_RECORDING_STOPPED = false;
-
 GRAD_REPLAY_SENDING_DELAY = 0.1;
-
-GRAD_REPLAY_SIDES = [west, east, civilian];
-// vehicle setVariable ["GRAD_replay_track", true];
-GRAD_REPLAY_AI_VEHICLES_TRACKED = false; 
-GRAD_REPLAY_AI_ONFOOT_TRACKED = false;
-
-
 GRAD_REPLAY_DATABASE_TEMP = [];
 GRAD_REPLAY_DATABASE = [];
 
-REPLAY_STEPS_PER_TICK = 1;
+// vehicle setVariable ["GRAD_replay_track", true];
+
+GRAD_REPLAY_SIDES = [(missionConfigFile >> "GRAD_Replay"), "trackedSides", [west, east, civilian]] call BIS_fnc_returnConfigEntry;
+GRAD_REPLAY_AI_VEHICLES_TRACKED = [(missionConfigFile >> "GRAD_Replay"), "trackedVehicles", false] call BIS_fnc_returnConfigEntry;
+GRAD_REPLAY_AI_ONFOOT_TRACKED = [(missionConfigFile >> "GRAD_Replay"), "trackedAI", false] call BIS_fnc_returnConfigEntry;
+REPLAY_STEPS_PER_TICK = [(missionConfigFile >> "GRAD_Replay"), "stepsPerTick", 1] call BIS_fnc_returnConfigEntry;
```

Comment:
Hier fehlt das Präfix und ich kanns nirgendwo sonst im Repo finden. Bewirkt das Settings irgendwas, oder ist das noch WIP, obwohls schon gemerged ist? 🤔

Reply:
Ich hätte es wissen müssen, dass Salbei seinen Shit net testet 😂

Also ja, ungetestet. Ich weiß nicht so richtig, was die sinnvolleren Vorgehensweise ist: Mergen und selbst testen oder PR verschimmeln lassen, bis Salbei wieder Arma-fähig is

Reply:
@nomisum das ist dein Käse, ich hab den nur übertragen. Für die Funktionalität bist du verantwortlich.

Reply:
Oha , war wohl ein Missverständnis. Dachte die Zeile ist ersatzlos rausgeflogen 😁

Reply:
Okay